pack/eoe/eoep.json

```diff
@@ -823,76 +1090,306 @@
+        "slot": "Arcane",
+        "text": "Uses (1 charge for each class among cards you control, including this one).\n[fast] Spend 1 charge and exhaust Close the Circle: Take a basic action. For each skill test performed during that action, you may use your [willpower] instead of the skill indicated for that action.",
+        "traits": "Ritual. Synergy.",
+        "type_code": "asset",
+        "xp": 1
+    },
+    {
+        "code": "08063",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "flavor": "May Oya's breath guide us all.",
+        "illustrator": "Pavel Kolomeyets",
+        "name": "Winds of Power",
+        "pack_code": "eoep",
+        "position": 63,
+        "quantity": 2,
+        "skill_willpower": 2,
+        "text": "Place 2 charges on an asset you conrol.\n[reaction] After you draw Winds of Power during your turn: Play it.",
+        "traits": "Spirit.",
+        "type_code": "asset",
+        "xp": 1
+    },
+    {
+        "code": "08064",
+        "cost": 0,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "illustrator": "Pavel Kolomeyets",
+        "name": "Foresight",
+        "pack_code": "eoep",
+        "position": 64,
+        "quantity": 2,
+        "skill_intellect": 2,
+        "text": "Fast. Play when an investigator at your location would draw a card from their deck or from the encounter deck.\nName a card. If the drawn card is the named card, that investigator may either (choose one):\n- Cancel that card's effects and discard it.\n- Immediately play that card at -2 cost.",
+        "traits": "Augury.",
+        "type_code": "event",
+        "xp": 1
+    },
+    {
+        "code": "08065",
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "flavor": "When knowledge is lacking, belief must suffice.",
+        "illustrator": "Mark Bulahao",
+        "name": "Occult Theory",
+        "pack_code": "eoep",
+        "position": 65,
+        "quantity": 2,
+        "text": "While Occul Theory is in your hand or committed to a skill test, it gainst [willpower] icons equal to your [intellect], and [intellect] icons equal to your [willpower].",
+        "traits": "Practiced. Expert.",
+        "type_code": "skill",
+        "xp": 1
+    },
+    {
+        "code": "08066",
+        "cost": 0,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "flavor": "\"If I said that I could tell you when you were going to die, would you want to know?\"",
+        "illustrator": "Alice Duke",
+        "name": "Parallel Fates",
+        "pack_code": "eoep",
+        "position": 66,
+        "quantity": 2,
+        "skill_wild": 1,
+        "skill_willpower": 1,
+        "text": "Look at the top 6 cards of an investigator's deck or the encounter deck. You may either return them to the top of that deck in any order, or shufflethem into that deck. If you looked at an investigator's deck, that investigatgor may draw 1 card.",
```

Comment:
"shuffle them" not "shufflethem"
"investigator" not "investigatgor" (at the end)